brainzutils/musicbrainz_db/serialize.py

```diff
@@ -85,7 +118,7 @@ def serialize_recording(recording, includes=None):
     if recording.video:
         data['video'] = True
 
-    if 'rating' in includes and includes['rating']:
+    if getattr(recording, 'rating', None):
```

Comment:
This is because in [utils.py](https://github.com/metabrainz/brainzutils-python/pull/81/files#diff-75345d2e7ede0b0e5b7e6a1f02fe4b04eaf287b12dad60d2bbd84cebb8b19a66L33) we only add `.rating` to the model object if there is a meta table row for it. This means that in the case that there's no rating, we could end up not adding it, which means that `recording.rating` would fail. Perhaps instead we could always add a .rating attr to this model.

Reply:
I did some experiments and also confirmed with @reosarevok on IRC that every entity has a row in corresponding meta table so rating will be added to all entities.

Reply:
Oh, my original comment was unclear. I wanted to say that replace `getattr` with `recording.rating`.

Reply:
good catch, I just assumed that meta rows would only exist if at least one rating had been added! I'll update this